ci-operator/templates/cluster-launch-e2e-upgrade.yaml

```diff
@@ -199,7 +199,7 @@ objects:
 
         ansible-playbook -vv \
           -e openshift_test_repo="${PREVIOUS_RPM_REPO}" \
-          -e 'oreg_url=${oreg_url}' \
+          -e oreg_url="${oreg_url}" \
           playbooks/${CLUSTER_TYPE}/openshift-cluster/launch.yml
```

Comment:
Is the `TYPE` env variable still required?

Either use `${TYPE}` with double quotes or drop it from the container's `env` and use `${CLUSTER_TYPE}` with single quotes.

Reply:
Its required for openshift-ansible's `entrypoint-provider` - https://github.com/openshift/openshift-ansible/blob/master/images/installer/root/usr/local/bin/entrypoint-provider#L28.

Replaced `${CLUSTER_TYPE}` with `"${TYPE}"` where possible